src/tests/ftest/container/query_attribute.py

```diff
@@ -34,17 +35,38 @@ class ContainerQueryAttributeTest(TestWithServers):
         that was returned when creating the pool/container.
 
         Attribute test:
-        1. Create 5 attributes to a container. We can set attribute
-        name and corresponding value, so set unique name-value pair for each of
-        the 5 attributes.
-        2. Call continaer list-attrs. This returns the 5 attribute names, so
-        compare them against the actual names used.
-        3. Call container get-attr. This returns the value of the corresponding
-        name, so do this for each of the 5 attributes.
+        1. Prepare 7 types of strings; alphabets, numbers, special characters,
+        etc.
+        2. Create attributes with each of these 7 types in attr and value;
+        i.e., 14 total attributes are created.
+        3. Call get-attr for each of the 14 attrs and verify the returned
+        values.
+        4. Call list-attrs and verify the returned attrs.
 
     :avocado: recursive
     """
 
+    def __init__(self, *args, **kwargs):
+        """Initialize a ContainerQueryAttribute object."""
+        super(ContainerQueryAttributeTest, self).__init__(*args, **kwargs)
+        self.expected_pool_uuid = None
+        self.expected_cont_uuid = None
+        self.sr = None
+        self.daos_cmd = None
+
+    def create_pool_container(self):
+        """Create a pool and a container in the pool.
+
+        Save some variables so that we can use them in the tests.
+        """
+        pool_create_result = self.get_dmg_command().get_output(
+            "pool_create", scm_size="1G")
+        self.expected_pool_uuid = pool_create_result[0]
+        self.sr = pool_create_result[1]
```

Comment:
If we should be also testing the destruction of the pool, we probably should be using `self.add_pool()` here (with a `pool/scm_size: 1G` test yaml entry) to set `self.pool` - which enables its destruction in `tearDown()`.  The `self.pool.uuid` and `self.pool.svcn` attributes will have have the values you need.

Reply:
Done